src/Events/SupervisorProcessDied.php

```diff
@@ -0,0 +1,36 @@
+<?php
+declare(strict_types=1);
```

Comment:
Remove this please. We don't use it in any Laravel code.

Reply:
Sorry, it's in my file template in my IDE. Removed!